tests/integration/validations/factory-dependent-keys-test.js

```diff
@@ -42,21 +56,26 @@ module(
       );
     });
 
-    test('ds-error validator creates correct dependent keys', function (assert) {
+    test('ds-error validator creates correct dependent keys', async function (assert) {
       this.owner.register('validator:ds-error', DSErrorValidator);
       this.owner.register('validator:length', LengthValidator);
 
       let DSErrorValidations = buildValidations({
         username: validator('ds-error'),
       });
-      let obj = setupObject(this, EmberObject.extend(DSErrorValidations), {
-        errors: Errors.create(),
-        username: '',
-      });
+
```

Comment:
I see other uses of `setupObject`, why does this one need changed and not the others?

Reply:
👌🏻 makes sense